src/taxonomy/taxonomy-menu/TaxonomyMenu.jsx

```diff
@@ -98,12 +98,19 @@ const TaxonomyMenu = ({
     </>
   );
 
+  const toggleProps = iconMenu ? {
+    as: IconButton,
+    src: MoreVert,
+    iconAs: Icon,
+  } : {
+    as: Button,
+  };
+
   return (
-    <Dropdown onToggle={(_isOpen, ev) => ev.preventDefault()}>
+    <Dropdown id="taxonomy-menu" onToggle={(_isOpen, ev) => ev.preventDefault()}>
```

Comment:
nit: since this menu might be shown multiple times on the TaxonomyList page, the `id` attributes used here won't be unique on the page (though they should be).

But if you include the `taxonomy.id` to the `id`, then they will be unique.

```suggestion
    <Dropdown id={`taxonomy-menu-${taxonomy.id}`} onToggle={(_isOpen, ev) => ev.preventDefault()}>
```